EventListener/VersionListener.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the xAPI package.
+ *
+ * (c) Christian Flothmann <christian.flothmann@xabbuh.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace XApi\LrsBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+/**
+ * @author Jérôme Parmentier <jerome.parmentier@acensi.fr>
+ */
+class VersionListener
+{
+    public function onKernelRequest(GetResponseEvent $event)
+    {
+        if (!$event->isMasterRequest()) {
+            return;
+        }
+
+        $request = $event->getRequest();
+
+        if (!$request->attributes->has('xapi_lrs.route')) {
+            return;
+        }
+
+        if (null === $version = $request->headers->get('X-Experience-API-Version')) {
+            throw new BadRequestHttpException('Missing required header "X-Experience-API-Version".');
+        }
+
+        if ($version === '1.0' || 0 === strpos($version, '1.0.')) {
```

Comment:
I don't see the point of normalizing its value (from `1.0` to `1.0.0`). We won't use this anywhere else.